sinks/types.go

```diff
@@ -19,7 +19,7 @@ import (
 	"fmt"
 )
 
-var argSink = flag.String("sink", "memory", "Backend storage. Options are [memory | influxdb | bigquery]")
+var argSink = flag.String("sink", "memory", "Backend storage. Options are [memory | influxdb | bigquery | gcm ]")
```

Comment:
Ok. We can clarify that in the Readme as well.

On Wed, Feb 18, 2015 at 12:31 PM, Victor Marmol <notifications@github.com>
wrote:

> In sinks/types.go
> <https://github.com/GoogleCloudPlatform/heapster/pull/112#discussion_r24937616>
> :
>
> > @@ -19,7 +19,7 @@ import (
> >  	"fmt"
> >  )
> >
> > -var argSink = flag.String("sink", "memory", "Backend storage. Options are [memory | influxdb | bigquery]")
> > +var argSink = flag.String("sink", "memory", "Backend storage. Options are [memory | influxdb | bigquery | gcm ]")
>
> Today we only support running in GCE (which we check and error out on in
> New) and we get the auth information from the machine's service accounts
> (which we also check and error out on).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/heapster/pull/112/files#r24937616>
> .
>